AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestMatrixStore.kt

```diff
@@ -158,7 +158,8 @@ internal class TestMatrixStore(
             projectId = firebaseProjectId,
             flakyTestAttempts = 2,
             testSpecification = TestSpecification(
-                disableVideoRecording = true,
+                testTimeout = "1800s",
```

Comment:
if last successful test is 28m, this should be a lot longer (maybe 60 min?)

Reply:
The max is unfortunately 45m for physical devices :/ I can set it to the max though.

Reply:
ugh i wonder that will bite us when we migrate androidX.

Reply:
Done.